vt-digital-libraries-platform/vt-digital-libraries-platform.github.io
# Adds post 2020-04-7-Multi-Site-Achieved

**Add new post to Github.io site.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2077) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Adds post to github.io site

# What's the changes? (:star:)
Adds post and upgrades jekyll gems, though I'm not sure the upgrade is desirable.  If not, someone please tell me how to avoid that.  I guess I could add those files to gitignore.


# How should this be tested?

A description of what steps someone could take to:
* Make sure it's accurate 
* Check to see if I typoed!

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* What branch to be used for testing this PR?
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository? 
* Could this change impact execution of existing code?

# Interested parties
@andreaWaldren 

(:star:) Required fields
